### PR TITLE
Alternative search validation.

### PR DESCRIPTION
**Pull request by Animesh-Ghosh**
Alternate fix for #26.

Now the search won't be submitted when the form is empty.

### CODE REVIEW

File: app/templates/home.html [hunk 1]
```diff
@@ -30,10 +30,10 @@ <h1 class="title" align="center">Search Tutorials</h1>
       <img src="{% static 'app/search.png' %}" height="100" width="100">
     </center>
     <br>
-    <form action="{% url 'app:search-results' %}" method="get">
+    <form action="{% url 'app:search-results' %}" method="get" onsubmit="return checkEmpty()">
       <div class="field">
         <div class="control has-icons-left">
-          <input class="input is-rounded" name="q" type="text" placeholder="search tutorials..." id="txt" onkeyup="manage(this)">
+          <input class="input is-rounded" name="q" type="search" placeholder="search tutorials..." id="search-bar">
```
Comment (Bhupesh-V) [inline app/templates/home.html]: the `id` of the search bar is updated but the `checkEmpty()` function is not updated accordingly

Comment (Animesh-Ghosh) [reply app/templates/home.html]: I have used `checkEmpty()` when the form is submitted. Check the onsubmit event of the form tag.

File: app/static/app/js/custom.js [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
-    if (txt.value != '') {
-        bt.disabled = false;
-    }
-    else {
-        bt.disabled = true;
-    }
+
+function checkEmpty() {
+	var input = document.getElementById("search-bar");
+	if (input.value == "" || input.value == null) {
```
Comment (Animesh-Ghosh) [inline app/static/app/js/custom.js]: Change '==' to '===' for good measure.

**Comment by Bhupesh-V** [comment]
Also update your branch, there is a conflict in `home.html`

File: app/static/app/js/custom.js (hunk 2 above)
Comment (Bhupesh-V) [reply app/static/app/js/custom.js]: OK

**Comment by Animesh-Ghosh** [comment]
> 
> 
> Also update your branch, there is a conflict in `home.html`

Should I resolve the conflict from the [web editor](https://github.com/Bhupesh-V/tutorialdb/pull/30/conflicts)? Only need to remove the `disabled` attribute.

**Pull request merged**